exercises/concept/rpn-calculator-inspection/.meta/exemplar.ex

```diff
@@ -0,0 +1,34 @@
+defmodule RPNCalculatorInspection do
+  def start_reliability_check(calculator, input) do
+    %{input: input, pid: spawn_link(fn -> calculator.(input) end)}
+  end
+
+  def await_reliability_check_result(%{pid: pid, input: input}, results) do
+    receive do
+      {:EXIT, ^pid, :normal} -> Map.put(results, input, :ok)
```

Comment:
In the match exercises, did we talk about pinning the value?

Reply:
It's only mentioned in `pattern-matching/about.md` (https://github.com/exercism/elixir/blob/9774308bb6b898df6ecbde64c5c6e48c651870ae/concepts/pattern-matching/about.md#L24), not in the exercise itself. I thought that + a hint in this exercise would be enough, given that this is optional and could instead be achieved with a guard.